addons/point_of_sale/models/res_partner.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.partner'
+
+    barcode = fields.Char(string='Barcode', help="BarCode", oldname='ean13')
```

Comment:
not sure where this barcode field comes from but it's definitely not present in the source code